### PR TITLE
[demo-app-ios] Fix color issues in Dark Mode

### PR DESCRIPTION
**Pull request by AnatolyPristensky**
* ~[ ] Has `CHANGELOG.md` been updated?~
* ~[ ] Are tests passing locally?~
* [x] Are the files formatted correctly?
* ~[ ] Did you add unit tests?~
* [x] Did you check UI tests on the sample app? They are not executed on CI.
* [x] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Fixed bug with incorrect rendering of Analytics tab with a DarkMode.
Also corrected incorrect rendering of MSEnumPicker wheel with a DarkMode.

## Related PRs or issues

[AB#87020](https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/87020)

### CODE REVIEW

File: Sasquatch/Sasquatch/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -1924,25 +1924,25 @@
...
                             <tableView clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="28" sectionFooterHeight="28" translatesAutoresizingMaskIntoConstraints="NO" id="usS-eK-aCf">
                                 <rect key="frame" x="16" y="146" width="343" height="464"/>
-                                <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                <color key="backgroundColor" systemColor="groupTableViewBackgroundColor"/>
                                 <prototypes>
                                     <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" insetsLayoutMarginsFromSafeArea="NO" selectionStyle="blue" hidesAccessoryWhenEditing="NO" indentationLevel="1" indentationWidth="0.0" id="ywr-BI-7Ti">
-                                        <rect key="frame" x="0.0" y="28" width="343" height="43.5"/>
+                                        <rect key="frame" x="0.0" y="24.5" width="343" height="43.5"/>
```
Comment (AnastasiaKubova) [inline Sasquatch/Sasquatch/Base.lproj/Main.storyboard:1937]: Do we really need these changes?

Comment (AnatolyPristensky) [reply Sasquatch/Sasquatch/Base.lproj/Main.storyboard:1937]: These changes have been added automatically by Xcode.
I've tried to discard them, but they appeared every time when I've opened a project.
Also I've discussed it with @DmitriyKirakosyan and he said that these changes can be kept.

**Review by DmitriyKirakosyan — DISMISSED** [review]

File: Sasquatch/Sasquatch/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -2008,7 +2008,7 @@
         <image name="distribute" width="23" height="23"/>
         <image name="fuel" width="23" height="23"/>
         <systemColor name="groupTableViewBackgroundColor">
-            <color red="0.94901960784313721" green="0.94901960784313721" blue="0.96862745098039216" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
```
Comment (DmitriyKirakosyan) [inline Sasquatch/Sasquatch/Base.lproj/Main.storyboard:2011]: Why do we need to change systemColor?

Comment (AnatolyPristensky) [reply Sasquatch/Sasquatch/Base.lproj/Main.storyboard]: You are correct. These changes are unnecessary. I've removed them.

**Review by DmitriyKirakosyan — APPROVED** [review]

**Review by AnastasiaKubova — APPROVED** [review]

**Review by DmitriyKirakosyan — APPROVED** [review]

**Pull request merged**